tests/specifications/polymorphic-required.yaml

```diff
@@ -0,0 +1,59 @@
+openapi: 3.0.1
```

Comment:
could you merge the yaml of these test files? 
I don't wanna separeate related tests.

Reply:
Sure, I merged those two into the existing `polymorphic.yaml` file now